src/gentropy/variant_to_gene.py

```diff
@@ -69,16 +66,10 @@ def __init__(
         # Read
         gene_index = GeneIndex.from_parquet(session, gene_index_path)
         vi = VariantIndex.from_parquet(session, variant_index_path).persist()
-        va = VariantAnnotation.from_parquet(session, variant_annotation_path)
+        # Reading VEP consequence to score table and cast the score to the right type:
         vep_consequences = session.spark.read.csv(
             vep_consequences_path, sep="\t", header=True
-        ).select(
-            f.element_at(f.split("Accession", r"/"), -1).alias(
-                "variantFunctionalConsequenceId"
-            ),
-            f.col("Term").alias("label"),
-            f.col("v2g_score").cast("double").alias("score"),
-        )
+        ).withColumn("score", f.col("score").cast("double"))
```

Comment:
When the score file is read, the schema not right. The scores need to be cast to double.